client/src/components/gallery_card/GalleryCard.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { FunctionComponent } from 'react';
+import { withTheme } from '@material-ui/core/styles';
+import styled from 'styled-components';
+import { useHistory } from 'react-router-dom';
+import CardMedia from '@material-ui/core/CardMedia';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import Box from '@material-ui/core/Box';
+import { motion } from 'framer-motion';
+
+import { Gallery } from "../../types/gallery"
+
+
+interface GalleryProps {
+  className?: string;
+  gallery: Gallery;
+}
+
+/**
+ * To make Gallery Card on Media Tap under Announcement.
```

Comment:
This doesn't make sense.
something like  `Display single card for an album on /media route`